catalog-entities/marketplace/packages/parfuemerie-douglas-scaffolder-backend-module-azure-repositorie.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: extensions.backstage.io/v1alpha1
 kind: Package
 metadata:
-  name: parfuemerie-douglas-scaffolder-backend-module-azure-repositories
+  name: parfuemerie-douglas-scaffolder-backend-module-azure-repositorie
```

Comment:
This is needed because the name is too long otherwise. It's the same as in 1.7 and earlier.